.circleci/config.yml

```diff
@@ -71,6 +71,9 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/
+      - run:
+          command: |
+            yarn add -W --dev react@^18.0.0-0 react-dom@^18.0.0-0 react-test-renderer@^18.0.0-0
```

Comment:
doesn't this mean nothing's being tested on react 17 anymore?

Reply:
This is only the code coverage task. Normal unit tests are run separately. I guess I could consolidate by doing a conditional check on the params to see if it should be reported...just don't want both tests to report themselves

Reply:
unit_tests are above. this is test_coverage